lib/hanami/components/components.rb

```diff
@@ -28,7 +28,13 @@ module Components # rubocop:disable Metrics/ModuleLength
       end
 
       resolve do |configuration|
-        Hanami::Logger.new(Hanami.environment.project_name, *configuration.logger) unless configuration.logger.nil?
+        if configuration.logger.is_a?(Array)
+          if configuration.logger.length == 1 && configuration.logger[0].is_a?(::Logger)
```

Comment:
Yes, I don't see any reason it wouldn't unless the internal usage of the logger in Hanami deviates from the standard `::Logger` interface. Otherwise, this should work great.